change/@azure-msal-browser-3eb1c367-abda-4451-b5e0-c6d2f2277136.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
No changes to public surface
```suggestion
  "type": "minor",
```

Reply:
I thought it ought to be major due to an additional mandatory param added to `NativeMessageHandler` constructor.

Reply:
Internal API changes are ok to be minor as the 1P library pins it's dependency to minor versions, not major.